spec/features/parks/03_parks_create_spec.rb

```diff
@@ -5,12 +5,14 @@
 # So that others can review it
 
 feature "authenticated users can view parks" do
+  let!(:user) { create(:user) }
   scenario "visitor signs in and views parks on index page" do
 
     visit root_path
 
```

Comment:
remove `visit root_path`, as it is redundant - you are running `sign_in_as` immediately after (which brings it to a new route)